state/sqlserver/migration.go

```diff
@@ -46,8 +46,8 @@ func newMigration(metadata *sqlServerMetadata) migrator {
 func (m *migration) newMigrationResult() migrationResult {
 	r := migrationResult{
 		itemRefTableTypeName:     fmt.Sprintf("[%s].%s_Table", m.metadata.Schema, m.metadata.TableName),
-		upsertProcName:           fmt.Sprintf("sp_Upsert_v3_%s", m.metadata.TableName),
-		getCommand:               fmt.Sprintf("SELECT [Data], [RowVersion] FROM [%s].[%s] WHERE [Key] = @Key AND ([ExpireDate] IS NULL OR [ExpireDate] > GETDATE())", m.metadata.Schema, m.metadata.TableName),
+		upsertProcName:           fmt.Sprintf("sp_Upsert_v4_%s", m.metadata.TableName),
+		getCommand:               fmt.Sprintf("SELECT [Data], [RowVersion], [ExpireDate] FROM [%s].[%s] WHERE [Key] = @Key AND ([ExpireDate] IS NULL OR [ExpireDate] > GETDATE())", m.metadata.Schema, m.metadata.TableName),
```

Comment:
should I remove `[ExpireDate]` here?

Reply:
No, I would leave it because then the stored procedure is different so when using 1.12, we need to create a new version. There's no harm in returning an extra value IMHO